config/config.yaml

```diff
@@ -9,3 +9,9 @@ sources:
   - "confident/"
 - id: "ghsa-malware"
   alias-id: true
+- id: "reversing-labs"
+  bucket: "s3://rl-github-ossf?awssdk=v2&profile=revlab&region=us-east-1"
+  prefixes:
+  - "RLMA-"
+  - "RLUP-"
```

Comment:
We actually use `RLUA-` for the update prefix.
Sorry for the confusion, I might have provided an example which had the RLUP prefix way back but we landed on `RLUA` in the end, which should have been communicated at some point I believe.

Reply:
Thanks, I've updated this.